README.md

```diff
@@ -1 +1,177 @@
-# To-Do-list
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠️ Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
```

Comment:
![image](https://github.com/peter-mwovi/To-Do-list/assets/93039359/7816f553-a9fc-4bb2-b06d-a07d545c19f8)

- [ ] It would be nice if you fix the **Deployment** link because right now it is broken :fearful: Kindly fix it or remove it to make your readme descriptive and professional :+1: